Argo/Globals/JSONDeserializer.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+import Runes
+
+public struct JSONDeserializer {
+  public static func deserialize(data: NSData) -> JSON? {
```

Comment:
_probably_. I think that's an easy enough refactor later on if we actually have real-world usage for parsing other sources of `NSData`